lua/cfc_loadouts/client/cl_cfc_loadout.lua

```diff
@@ -1,22 +1,20 @@
 --local UICOLOR = Color( 36, 41, 67, 255 )
-
-local weaponCategorised = {}
-local allWeapons = {}
-
-file.CreateDir("cfc_loadout")
-
-hook.Add( "InitPostEntity", "CFC_Loadouts_ReadyClientCheck", function()
-    net.Start( "CFC_Loadout_InitialSpawn" )
-    net.SendToServer()
-end )
-
-net.Receive( "CFC_Loadout_SendRestrictions", function()
+local weaponCategorised = { }
+local allWeapons = { }
+file.CreateDir( "cfc_loadout" )
+--WIPd
+--hook.Add( "InitPostEntity", "CFC_Loadouts_ReadyClientCheck", function( )
+net.Start( "CFC_Loadout_InitialSpawn" )
+net.SendToServer( )
+
+--end )
```

Comment:
Are you sure this is ready at this point? There are a few ways to handle this. Usually people use `PlayerInitialSpawn` with a delay on serverside. Clients aren't ready for networking at that point, so a delay of some kind is necessary.

More info in the `bug` section here https://wiki.facepunch.com/gmod/GM:PlayerInitialSpawn

Reply:
The hook is disabled because i'd break in rapid update testing, enabling the hook works fine.